src/components/elements/boxes.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+
+    <component class="ods-container__link"
+        :is="componentId"
+        v-bind="attributId">
+
+        <div class="ods-container__box"
+            :class="sizeBox">
+            
+            <div class="ods-container__box-icon">
+                
+                <img :class="sizeIcon"
+                    :src="img"
+                    alt="OpenDataSoft Documentation"/>
+
+            </div>
+            
+            <div class="ods-container__box-desc">
+                
+                <h2 class="ods-container__box-desc-h2"
+                    id="index-title-platform">
+                    {{ trad.title[lang] }}
+                </h2>
+
+                <p class="ods-container__box-desc-p"
+                    id="index-desc-platform"
+                    v-html=trad.desc[lang]>
+                </p>
+
+            </div>
+
+        </div>
+
+    </component>
+
+</template>
+
+<script>
+export default {
+    name: 'boxes',
+    data: function () {
+        if (this.trad.url[this.lang].indexOf('https') > -1 ||
+            this.trad.url[this.lang].indexOf('http') > -1) {
+            return {
+                componentId: 'a',
+                attributId: { href : this.trad.url[this.lang] }
+            }
+        }
+        else {
+            return {
+                    componentId: 'router-link',
+                    attributId: { to : this.trad.url[this.lang] }
+                }
+        }
+    },
+    props: {
+        lang: String,
+        trad: Object,
+        img: String,
+        sizeBox: String,
+        sizeIcon: String
+    }
+}
+</script>
+
+<style lang='less'>
+@import "../../assets/less/variables";
+@import "../../assets/less/components";
+
+.ods-container__link {
+    display: block;
+}
+
+.ods-container__box {
+    &:extend(.default-box);
+    @media (max-width: @box-max-width) {
+        width: 90%;
+    }
+    @media (min-width: @box-max-width) and (max-width: @mobile-width) {
+        width: 330px;
+    }
+    @media (min-width: @desktop-width) {
+        width: 330px;
+        &:hover {
+            background-color: white;
+            border: 1px solid lighten(@grey-border, 40%);
+            -webkit-box-shadow: 0px 10px 26px 5px lighten(@grey-border, 15%);
+            -moz-box-shadow: 0px 10px 26px 5px lighten(@grey-border, 15%);
+            box-shadow: 0px 10px 26px 5px lighten(@grey-border, 15%);
+        }
+    }
+}
+
+.ods-container__box-index {
```

Comment:
Maybe a `min-height` could be a better fit here? Since the boxes contain text that will be displayed in multiple languages (and therefore different lengths of text which could overflow the box), you should try to avoid such fixed values and use something more flexible.